src/engine/client/client.cpp

```diff
@@ -865,7 +865,7 @@ int CClient::UnpackServerInfo(CUnpacker *pUnpacker, CServerInfo *pInfo, int *pTo
 
 	// don't add invalid info to the server browser list
 	if(pInfo->m_NumClients < 0 || pInfo->m_NumClients > MAX_CLIENTS || pInfo->m_MaxClients < 0 || pInfo->m_MaxClients > MAX_CLIENTS ||
-		pInfo->m_NumPlayers < 0 || pInfo->m_NumPlayers > pInfo->m_NumClients || pInfo->m_MaxPlayers < 0 || pInfo->m_MaxPlayers > pInfo->m_MaxClients)
+		pInfo->m_NumPlayers < 0 || pInfo->m_NumPlayers > pInfo->m_NumClients || pInfo->m_MaxPlayers < 0 || pInfo->m_MaxPlayers > pInfo->m_MaxClients || pInfo->m_MaxPlayers > MAX_PLAYERS)
```

Comment:
To my understanding, the drop should only apply to covert mods who broadcast as pure gametypes:

> modified DM/TDM/CTF/... servers 

Those are already forbidden and manually banned from the master servers, but it's only servers that have `pInfo->m_MaxPlayers > MAX_PLAYERS` **and** broadcast as DM/TDM/CTF/SUR/LMS.

Reply:
I understood the later, drop anything with MaxPlayer > 16, whatever the gametype.
Both are fine with me. @oy ?

Reply:
Upon re-reading it became clearer that you are right, @nheir, sorry, I'll resolve.

Reply:
Yeah, that's correct. As we keep on having a maximum player number of 16 and don't want to touch anything that relies on that (scoreboard etc.), the client should stay away from things that compromise that.